### ISSUE TITLE
[asking some question about new release code]

### ISSUE THREAD

**Issue by akimo12345** [davidsandberg/facenet#696]
Hi David:
               I follow your new release code and then got something confused. can you share the reference? many thanks for your helping.
               Q1: why do you use norm loss instead center loss ?  
               Q2: why img use fixed standardization --> img - 127.5 / 128.0, 127.5 is mean value? 128.0 is stddev?  tf.image.per_image_standardization is not better than fixed std?
BR,
Joseph

**Comment by billtiger** [comment]
hello @akimo12345 
about Q1,i think david add center loss also,there are using norm loss and center loss.
and i have same question about Q2.

**Comment by akimo12345** [comment]
Hi, billtiger
                thanks for your reply. Both of two factor(center_loss_factor, prelogits_norm_loss_factor) are set to zero on vggface dataset training command, so that I think loss function only has softmax_cross_entropy, not included one of them.

**Comment by whuguozili** [comment]
Q1: maybe they made some experiments on center loss. however, the results were not satisfying.
Q2: the standardization is stretching the RGB values from [0, 255] to [-1, 1]. just a linear transformation.
